bindings/wallet-cordova/src/android/WalletPlugin.java

```diff
@@ -120,6 +121,9 @@ public boolean execute(final String action, final CordovaArgs args, final Callba
         case "SETTINGS_GET":
             settingsGet(args, callbackContext);
             break;
+        case "FRAGMENT_ID":
```

Comment:
Shouldn't other new methods be exposed as well?

Reply:
Well, in terms of having a proper API perhaps we should, but I think making it right would require to change all the functions that return transactions as binaries to return a `Fragment` handler instead, which then you can get the id or the bytes (and have to deallocate).

As that's a breaking change and more work (and I don't think it's worth doing it right now), I just skipped that part and went from binary fragment to binary id directly, because it seems more consistent to me.